core/types/src/report.rs

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! SGX Report
+
+use crate::{
+    impl_newtype_for_bytestruct, new_type_accessors_impls, Attributes, ConfigSvn, CpuSvn, IsvSvn,
+    MiscellaneousSelect,
+};
+use mc_sgx_core_sys_types::{
+    sgx_config_id_t, sgx_isvext_prod_id_t, sgx_isvfamily_id_t, sgx_measurement_t, sgx_prod_id_t,
+    sgx_report_body_t, sgx_report_data_t, SGX_CONFIGID_SIZE, SGX_HASH_SIZE,
+    SGX_ISVEXT_PROD_ID_SIZE, SGX_REPORT_BODY_RESERVED1_BYTES, SGX_REPORT_BODY_RESERVED2_BYTES,
+    SGX_REPORT_BODY_RESERVED3_BYTES, SGX_REPORT_BODY_RESERVED4_BYTES, SGX_REPORT_DATA_SIZE,
+};
+
+/// A measurement
```

Comment:
Up for suggestions on these transitive types.  Don't think it's ideal they're here but still flushing out where else they might be used.